zerver/openapi/zulip.yaml

```diff
@@ -20463,7 +20463,30 @@ paths:
                 contentType: application/json
       responses:
         "200":
-          $ref: "#/components/responses/SimpleSuccess"
+          description: |
+            A success response containing the unique ID of the user group
+            with the key `group_id`. This field provides a straightforward
+            way to reference the newly created user group.
+
+            **Changes**: New in Zulip 10.0 (feature level 316).
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/JsonSuccessBase"
+                  - required:
+                      - group_id
+                    additionalProperties: false
+                    properties:
+                      result: {}
+                      msg: {}
+                      ignored_parameters_unsupported: {}
+                      group_id:
+                        type: integer
+                        description: |
+                          The unique ID of the created user groups.
```

Comment:
"created user group" -- plural is wrong here. One should always carefully review text in any documentation you write, since it looks very unprofessional to have typos in docs.